modules/recon/credentials-credentials/hashes_org.py

```diff
@@ -9,24 +9,26 @@ class Module(BaseModule):
         'name': 'Hashes.org Hash Lookup',
         'author': 'Tim Tomes (@LaNMaSteR53) and Mike Lisi (@MikeCodesThings)',
         'version': '1.0',
-        'description': 'Uses the Hashes.org API to perform a reverse hash lookup. Updates the \'credentials\' table with the positive results.',
+        'description': 'Uses the Hashes.org API to perform a reverse hash lookup. Updates the \'credentials\' table '
```

Comment:
Was adding all of these line breaks necessary? Does it even work this way? If it does, they must have changed this in Python 3. Anyway, probably not necessary to wrap strings that short. Just an OCD thing for me.